streams/src/main/java/org/apache/kafka/streams/state/internals/NullableValueAndTimestampSerde.java

```diff
@@ -48,42 +45,4 @@ public NullableValueAndTimestampSerde(final Serde<V> valueSerde) {
         );
     }
 
-    static final class BooleanSerde {
```

Comment:
Are there tests for this code that need to cleaned up?

Reply:
I think not, I've found following unit tests that are using NullableValueAndTimestampSerializer (currently the only code that depends on BooleanSerde):

- NullableValueAndTimestampSerdeTest - These unit-tests check everything as a whole class. There is not a single unit-test that tests only BooleanSerde.
- MeteredVersionedKeyValueStoreTest - Same as above.